backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/input/restapi/schema/PreviewRequestSchema.java

```diff
@@ -0,0 +1,17 @@
+package de.bund.digitalservice.ris.norms.adapter.input.restapi.schema;
+
+import java.util.Map;
+import lombok.Getter;
+
+/** Schema for the request for rendering the preview of a norm xml. */
+@Getter
+public class PreviewRequestSchema {
+  /** The xml of the norm. */
+  private String norm;
+
+  /**
+   * The xmls of norms which are referenced by the norm (e.g. in passiveModifications) and should
+   * not be loaded from the database.
+   */
+  private Map<String, String> customNorms = Map.of();
```

Comment:
Do we need to have a Map with pairs of ELI + XML or is the XML just enough?

Reply:
Oh true, we can get the eli from the xml so no need for a map: a Set/Array/List is totally enough, will change that!
I would like to support sending multiple custom norms to keep this endpoint general as norms can reference multiple other norms and to keep the possibility to preview a completely custom combination of norms. (But feel free to challenge that)

Reply:
I agree. The time machine will need this functionality someday when we generate the ZF1 and ZF2 because we could have the case that a ZF0 contains mods from different amending laws